Gruntfile.js

```diff
@@ -0,0 +1,112 @@
+"use strict";
+const webpack = require("webpack");
+const webpackConfig = require("./webpack.config");
+const merge = require("webpack-merge");
+
+const webpackConfigRelease = webpackConfig.map(config => merge(config, {
+    devtool: false,
+    mode: "production",
+    optimization: { minimize: true }
+}));
+
+module.exports = function(grunt) {
+    const pkg = grunt.file.readJSON("package.json");
+    grunt.initConfig({
+
+        watch: {
+            updateWidgetFiles: {
+                files: [ "./src/**/*" ],
+                tasks: [ "webpack:develop", "compress:dist", "copy:distDeployment", "copy:mpk" ],
```

Comment:
`copy` is being miss-used simply do `copy`

Reply:
done